nexuse2e-webapp/src/main/frontend/src/app/pipes/caps-to-title-case.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { TitleCasePipe } from "@angular/common";
+
+@Pipe({
+  name: "capsToTitleCase",
+  pure: true,
+})
+export class CapsToTitleCasePipe implements PipeTransform {
+  transform(value: unknown): unknown {
+    if (typeof value === "string") {
+      const titleCasePipe = new TitleCasePipe();
+      const uppercase = value.toUpperCase();
+      if (value === uppercase) {
+        return titleCasePipe.transform(value) as string;
```

Comment:
We need the TitleCasePipe only if the string is uppercase, so we can initialize it only in this case.
```suggestion
      const uppercase = value.toUpperCase();
      if (value === uppercase) {
        const titleCasePipe = new TitleCasePipe();
        return titleCasePipe.transform(value) as string;
```

Reply:
Done.